### DISCUSSION TITLE
Bash Script and Inventory

### DISCUSSION THREAD

**Discussion by buzzzo** [semaphoreui/semaphore#2559] [Q&A] [answered]
I'm playing around semaphore to use as replacement of other automation tools especially for bash scripts.

What i'm expecting is that by creating a bask task script i will be able to run it against an inventory with credentials, but on the task definition i cannot find any option for specifying inventory.

The question is: Is this Behavior by default ? Is it meant to be executed ONLY on local side ? (where local = server where semaphore is running).

Thx in advance

**Comment by fiftin** [comment, marked as answer]
yes, it is true.

**Reply by buzzzo** [reply]
Thx for your answer.
Is there some workaround in order to run a bash script against an inventory ?

**Reply by fiftin** [reply]
I'm not sure it is required. It is duplicates of Ansible functionality. It will not work as reliably as Ansible (or will require a lot of work).

We have Runners. You can use them for remote running scripts.

**Reply by buzzzo** [reply]
Thx I will take a look
